homeassistant/components/technove/strings.json

```diff
@@ -0,0 +1,62 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "step": {
+      "user": {
+        "description": "Set up your TechnoVE station to integrate with Home Assistant.",
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]"
+        },
+        "data_description": {
+          "ip_address": "Hostname or IP address of your WLED device."
```

Comment:
🤔

Reply:
Oh no! That's embarassing. That's a leftover from a copy paste that I missed. Sorry, fixed.